dss/api/subscriptions_v2.py

```diff
@@ -29,16 +31,21 @@ def get(uuid: str, replica: str):
 @security.authorized_group_required(['hca', 'public'])
 def find(replica: str):
     owner = security.get_token_email(request.token_info)
-    subs = [s for s in get_subscriptions_for_owner(Replica[replica], owner) if owner == s['owner']]
-    for s in subs:
-        s['replica'] = Replica[replica].name
-        if 'hmac_secret_key' in s:
-            s.pop('hmac_secret_key')
-    return {'subscriptions': subs}, requests.codes.ok
+    subscriptions = get_subscriptions_for_owner(Replica[replica], owner)
+    for subscription in subscriptions:
+        subscription['replica'] = Replica[replica].name
+        if 'hmac_secret_key' in subscription:
+            subscription.pop('hmac_secret_key')
+    return {'subscriptions': subscriptions}, requests.codes.ok
 
 
 @security.authorized_group_required(['hca', 'public'])
 def put(json_request_body: dict, replica: str):
+    owner = security.get_token_email(request.token_info)
+    if count_subscriptions_for_owner(Replica[replica], owner) > SUBSCRIPTION_LIMIT:
+        raise DSSException(requests.codes.not_acceptable, "not_acceptable",
```

Comment:
Does this error code need to be included in `dss-api.yml`?

Reply:
Good point.  Will update.